exchanges/kline/kline_test.go

```diff
@@ -241,16 +249,123 @@ func TestKlineErrors(t *testing.T) {
 }
 
 func TestTotalCandlesPerInterval(t *testing.T) {
-	end := time.Now()
-	start := end.AddDate(-1, 0, 0)
+	start := time.Date(2019, 1, 1, 0, 0, 0, 0, time.UTC)
+	end := time.Date(2019, 12, 31, 23, 59, 59, 59, time.UTC)
```

Comment:
How about 
```
	start := time.Date(2019, 1, 1, 0, 0, 0, 0, time.UTC)
	end := time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC)
```
To show that oneYear can calculate properly? Plus it makes all the other numbers look prettier, but that's not really relevant 😄

Reply:
yeah good point

change has been made and pushed